server/routes/kanban.js

```diff
@@ -11,13 +10,18 @@ router.get('/', (req, res) => {
       return res.json(cards);
     })
     .catch(err => {
+<<<<<<< HEAD
       res.json(res.detail);
+=======
```

Comment:
You still have merge conflicts in this file. Choose which one of these works with your code now.